Student-Service/src/main/java/tw/waterball/judgegirl/studentservice/domain/usecases/DeleteStudentFromGroupUseCase.java

```diff
@@ -0,0 +1,29 @@
+package tw.waterball.judgegirl.studentservice.domain.usecases;
+
+import lombok.AllArgsConstructor;
+import tw.waterball.judgegirl.commons.exceptions.NotFoundException;
+import tw.waterball.judgegirl.entities.Group;
+import tw.waterball.judgegirl.studentservice.domain.repositories.GroupRepository;
+import tw.waterball.judgegirl.studentservice.domain.repositories.StudentRepository;
+
+import javax.inject.Named;
+
+/**
+ * @author - wally55077@gmail.com
+ */
+@Named
+@AllArgsConstructor
+public class DeleteStudentFromGroupUseCase {
+
+    private final StudentRepository studentRepository;
+
+    private final GroupRepository groupRepository;
+
+    public void execute(int groupId, int studentId)
+            throws NotFoundException {
+        Group group = groupRepository.findGroupById(groupId)
```

Comment:
add a comment above TODO: improve performance, should only perform one SQL query